tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -121,8 +130,17 @@ spec:
             indexImageResolved="$(jq -r '.index_image_resolved' <<<  "${build}")"
             newCatalogCreatedDate="$(date --date "$(skopeo inspect --config "docker://${indexImageResolved}" | \
               jq -r .created)" "+%s")"
-            upstreamCatalogCreatedDate="$(date --date "$(skopeo inspect --config "docker://$(params.targetIndex)" | \
-              jq -r .created)" "+%s")"
+
+            # authentication is only required for the targetIndex
+            create_auth_file
+            targetIndexCreated="$(skopeo inspect --config "docker://$(params.targetIndex)" | jq -r .created)"
+            # an empty targetdIndexCreated will cause `date` to use an arbitrary date that could allow
```

Comment:
I don't think this comment is necessary. Clearly it doesn't make sense to create a date out of an empty string - we don't need to know what side effect it would have.

Or maybe change it to something like: "If we cannot determine the target index created date, stop here (and force a new build)"

Reply:
fixed.